app/controllers/oauth/applications_controller.rb

```diff
@@ -109,12 +121,49 @@ def set_user
     end
 
     def app_params
-      params.require(:doorkeeper_application).permit(
-        :name, :redirect_uri, :scopes, :email_subject_prefix, :lead_application_source,
-        :email_from_address, :confidential,
-        :can_access_private_user_data, :can_find_or_create_accounts, :can_message_users,
-        :can_skip_oauth_screen,
-      )
+      if @user.is_administrator?
+        params.require(:doorkeeper_application).permit(
+          :name, :redirect_uri, :scopes, :email_subject_prefix, :lead_application_source,
+          :email_from_address, :confidential,
+          :can_access_private_user_data, :can_find_or_create_accounts, :can_message_users,
+          :can_skip_oauth_screen, :member_ids
```

Comment:
IIRC, `:member_ids` was outside of the nested `:doorkeeper_application` parameters?  If so I don't think using it here is doing what is desired since `params.require(:doorkeeper_application)` returns just the parameters nested under `:doorkeeper_application`.

Reply:
Removed member_ids